index.html

```diff
@@ -22,40 +23,78 @@ <h1>G4 Spell Checker Machine</h1>
+                                        <a id = 'df' class = "button">Download file</a>
+                                        <a id = 'prt' class = "button">Print</a>
+
                                 </div>
         
                                 <!-- The text area to collect the texts to be checked -->
                                 <div class="main">
-                                        <select name="lang">
-                                                <option value="en"> English</option>
-						<option value="ir"> Irish</option>
-                                        </select>
-                                        <div class="text-body" contenteditable="true" spellcheck="false" @input="event => onInput(event)" 
-                                                data-placeholder="Type or paste your text here or upload document <br> Clóscríobh nó greamaigh do théacs anseo nó uaslódáil doiciméad " v-html="richText">
+								<div>
+								<a href="#english" class = 'button' data-reload>English</a>
+								<a href="#irish" class = 'button' data-reload>Irish</a>
+								</div>
+								<br>
+                                 <div class="text-body" id = "mainContentBox" contenteditable="true" spellcheck="false" @input="event => onInput(event)" 
+                                                data-placeholder="Type or paste your text here or upload document!" v-html="richText">
                                         </div>
 
-	                                <button @click="checkTextGrammar">Check Spelling</button>
+	                                <button @click="checkTextGrammar" id = 'CSbutton'>Check Spelling</button>
                                 </div>
         
                                 <!-- The correction area to display the suggested texts -->
                                 <div class="right" >
-                                        <p id="suggest-text">Corrected suggestions/Moltaí ceartaithe</p>
+                                        <p id="suggest-text">Corrected suggestions</p>
                                         <ul>
                                                 <li v-for=" (word, index) in correctedWords"> {{ word.correct }} <button
-                                                @click="replaceWord(word)">replace</button></li>
+                                                @click="replaceWord(word)" id = 'rpl' >replace</button></li>
                                         </ul>
                                 </div>
                         </div>
         
                         <!-- The footer of the HTML page -->
-                        <div class="footer">© copyright Group-4 CSCI-5030 2021</div>
+                        <div class="footer" id ='ftr'>© copyright Group-4 CSCI-5030 2021</div>
                 </div>
                         <!--Java Script-->
                         <script>
+						
+						<!-- Script for Multi-Lingual Frontend -->
```

Comment:
Awesome! I can see that this is the JavaScript code that will switch the HTML page between English and Irish Language. Nice!